.github/workflows/release-please.yml

```diff
@@ -1,20 +1,17 @@
----
+name: Run release-please
+
 on:
-  push:
-    branches: [master]
+  push: {branches: master}
 
 permissions:
   contents: write
   pull-requests: write
 
-name: Run release-please
-
 jobs:
   release-please:
     runs-on: ubuntu-latest
     steps:
       - uses: googleapis/release-please-action@v4
         with:
           release-type: rust
-          package-name: ${{ github.event.repository.name }}
```

Comment:
@joelazar this is the only real change we care about in this PR and want to test out once merged.

Reply:
alrighty, hope it works 🤞